src/raw.rs

```diff
@@ -67,11 +59,20 @@ pub fn read_analysis_incremental<L: AnalysisLoader>(
                 }
 
                 let path = p.join(&l.name);
-                let is_fresh = timestamps.get(&path).map_or(true, |t| time > t);
-                if is_fresh {
-                    read_crate_data(&path)
-                        .map(|a| result.push(Crate::new(a, *time, Some(path))));
-                }
+                // TODO: Bring back path-based timestamps?
+                // This would speed up a bit reloading time, since non-blacklisted
+                // can be older than the ones loaded, so don't waste time
+                // reading the file and deserializing
```

Comment:
Yeah, it would be nice if we could avoid reading the file - reading and deserialising can be a significant overhead for large analysis files.

Reply:
@nrc Can I add this after this PR lands?

Reply:
Sure, that's fine.